views/signedup.tmpl

```diff
@@ -66,7 +66,7 @@
         <br>
         <h1 class="text-center">Come see us!</h1>
         <br>
-        <p class="text-center">We have a calendar of <a href="https://goo.gl/5APK7Q">all our meetings</a>, to which new members are always welcome, and you can drop by the station almost any time and there'll be someone to talk to.</p>
+        <p class="text-center">We have a calendar of <a href="/meetings.ics">all our meetings</a>, to which new members are always welcome, and you can drop by the station almost any time and there'll be someone to talk to.</p>
```

Comment:
will this give people an easy to read format, or just an integration?? also, this page only shows up once in someone's ury lifetime, so it maybe isn't best on the main website and keep it to myradio